### PR TITLE
Removed edge://flags step from "Get started with remote debugging Windows devices"

### PR DESCRIPTION
**Pull request by mikehoffms**
Removed the discussion and screenshot of "edge://flags" from section 2, and kept it in section 1.

To review the PR: Find "**edge://flags**" 2x in the **original article**:
https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/remote-debugging/windows
Section "Step 1: Set up the host (debuggee machine)" contains "Enable remote debugging through Windows Device Portal" - this setting is found in recent Edge.  Kept in article.
Section " Step 2: Set up the client (debugger machine)" contains "Enable remote Windows device debugging in edge://inspect" - this setting is not found in recent Edge.  Deleted from article.

**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/remote-debugging/windows?branch=pr-en-us-1563
Find "**edge://flags**" 1x now.

**Changes made in this PR:**
* In section "Step 2", removed the step for setting the edge://flags, flag: "Enable remote Windows device debugging in edge://inspect", article: "Get started with remote debugging Windows devices".
* Wordsmithing.
* Retagging, eg numbered the action steps.
* Tagging cleanup of images.

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/remote-debugging/windows.md [hunk 1]
```diff
@@ -28,13 +28,9 @@ The host or debuggee machine is the Windows 10 or later device that you want to
 
 ### Install and configure Microsoft Edge
 
-If you have not already, install Microsoft Edge from [this page](https://www.microsoft.com/edge).  If you are using a pre-installed version of Microsoft Edge on the host (debuggee) machine, verify that you have Microsoft Edge and not Microsoft Edge (EdgeHTML).  A quick way to check is to load `edge://settings/help` in the browser and confirm that the version number is 75 or higher.
+If you haven't already, install Microsoft Edge from [this page](https://www.microsoft.com/edge).  If you are using a pre-installed version of Microsoft Edge on the host (debuggee) machine, verify that you have Microsoft Edge (Chromium) and not Microsoft Edge (EdgeHTML).  A quick way to check is to load `edge://settings/help` in the browser and confirm that the version number is 75 or higher.
 
-Now navigate to `edge://flags` in Microsoft Edge.  In **Search flags**, type in **Enable remote debugging through Windows Device Portal**.  Set that flag to **Enabled**.  Then, choose the **Restart** button to restart Microsoft Edge.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/remote-debugging/windows.md:33]: @bwalderman - I removed this step (but kept the image).  Should this step in this section be retained?

File: microsoft-edge/devtools-guide-chromium/remote-debugging/windows.md [hunk 2]
```diff
@@ -43,66 +39,68 @@ Install the [Remote Tools for Microsoft Edge (Beta)](https://www.microsoft.com/s
...
+
+1.  To set up the client (debugger) machine, install Microsoft Edge from [this page](https://www.microsoft.com/edge) if you haven't already.
 
-To set up the client (debugger) machine, install Microsoft Edge from [this page](https://www.microsoft.com/edge) if you have not already.  If you are using a pre-installed version of Microsoft Edge on the host (debuggee) machine, verify that you have Microsoft Edge and not Microsoft Edge (EdgeHTML).  A quick way to check is to load `edge://settings/help` in the browser and confirm that the version number is 75 or higher.
+1.  If you're using a pre-installed version of Microsoft Edge on the host (debuggee) machine, verify that you have Microsoft Edge (Chromium) and not Microsoft Edge (EdgeHTML).  A quick way to check is to load `edge://settings/help` in the browser and confirm that the version number is 75 or higher.
 
-Now navigate to `edge://flags` in Microsoft Edge.  In **Search flags**, type in **Enable remote Windows device debugging in edge://inspect**.  Set that flag to **Enabled**.  Then, choose the **Restart** button to restart Microsoft Edge.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/remote-debugging/windows.md:83]: @bwalderman - I removed this step (in Step 2 section), and removed the screenshot.  This is the motivation for this PR.

File: microsoft-edge/devtools-guide-chromium/remote-debugging/windows.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/remote-debugging/windows.md:33]: This is what I don't understand about GitHub PR UI: When I update a Comment on **Files Changed** tab, it doesn't update here in the **Conversations** tab.  I retained, not removed, this step.

**Review by zoherghadyali — APPROVED** [review]

File: microsoft-edge/devtools-guide-chromium/remote-debugging/windows.md (hunk 1 above)
Comment (bwalderman) [reply microsoft-edge/devtools-guide-chromium/remote-debugging/windows.md:33]: LGTM

File: microsoft-edge/devtools-guide-chromium/remote-debugging/windows.md (hunk 2 above)
Comment (bwalderman) [reply microsoft-edge/devtools-guide-chromium/remote-debugging/windows.md:83]: LGTM

**Review by bwalderman — APPROVED** [review]

**Pull request merged**